ui/admin/angular.json

```diff
@@ -27,6 +27,8 @@
               "src/assets"
             ],
             "styles": [
+              "./node_modules/bootstrap/dist/css/bootstrap.css",
```

Comment:
Bootstrap is already include in the `src/index.html` file for developing purpose. With this inclusion, as angular dependency, there is some conflicts with the bootstrap bundles dependency. So this isn't needed.

Reply:
Changed